isis/src/base/apps/caminfo/caminfo.cpp

```diff
@@ -253,37 +277,15 @@ namespace Isis{
           CameraStatistics stats(filename, sinc, linc);
           Pvl camPvl = stats.toPvl();
 
-          PvlGroup cg = camPvl.findGroup("Latitude", Pvl::Traverse);
-          camstats->append(MakePair("MinimumLatitude", cg["latitudeminimum"][0]));
-          camstats->append(MakePair("MaximumLatitude", cg["latitudemaximum"][0]));
-
-          cg = camPvl.findGroup("Longitude", Pvl::Traverse);
-          camstats->append(MakePair("MinimumLongitude", cg["longitudeminimum"][0]));
-          camstats->append(MakePair("MaximumLongitude", cg["longitudemaximum"][0]));
-
-          cg = camPvl.findGroup("Resolution", Pvl::Traverse);
-          camstats->append(MakePair("MinimumResolution", cg["resolutionminimum"][0]));
-          camstats->append(MakePair("MaximumResolution", cg["resolutionmaximum"][0]));
-
-          cg = camPvl.findGroup("PhaseAngle", Pvl::Traverse);
-          camstats->append(MakePair("MinimumPhase", cg["phaseminimum"][0]));
-          camstats->append(MakePair("MaximumPhase", cg["phasemaximum"][0]));
-
-          cg = camPvl.findGroup("EmissionAngle", Pvl::Traverse);
-          camstats->append(MakePair("MinimumEmission", cg["emissionminimum"][0]));
-          camstats->append(MakePair("MaximumEmission", cg["emissionmaximum"][0]));
+          // Skips first "User Parameters" group.
+          for (int i = 1; i < camPvl.groups(); i++) {
+            PvlGroup &group = camPvl.group(i);
 
-          cg = camPvl.findGroup("IncidenceAngle", Pvl::Traverse);
-          camstats->append(MakePair("MinimumIncidence", cg["incidenceminimum"][0]));
-          camstats->append(MakePair("MaximumIncidence", cg["incidencemaximum"][0]));
-
-          cg = camPvl.findGroup("LocalSolarTime", Pvl::Traverse);
-          camstats->append(MakePair("LocalTimeMinimum", cg["localsolartimeMinimum"][0]));
-          camstats->append(MakePair("LocalTimeMaximum", cg["localsolartimeMaximum"][0]));
-
-          cg = camPvl.findGroup("ObliqueResolution", Pvl::Traverse);
-          camstats->append(MakePair("ObliqueResolutionMinimum", cg["ObliqueResolutionMinimum"][0]));
-          camstats->append(MakePair("ObliqueResolutionMaximum", cg["ObliqueResolutionMaximum"][0]));
```

Comment:
These need to remain as well.

They all should be tagged for removal in a future version as suggested earlier - or not.